### PR TITLE
p2p/simulation: fix p2p simulation framework

### PR DESCRIPTION
**Pull request by rjl493456442**
This PR does these fixes and improvements:
- Fix the `exec` adapter in p2p simulation
   - Remove the `ws://` prefix from the status endpoint, the `ws://` is already included in the `stack.WSEndpoint()` natively.
   - Don't register the services again in the node start. The registration is done in the initialization stage
   - Expose `admin` namespace via websocket. This namespace is necessary for connecting the peers via websocket

- Offer logging relevant options for `exec` adapter
   It's really painful to mix all log output in the single console. So this PR offers two additional options for `exec` adapter in this case testers can config the log output(e.g. file output) and log level for each p2p node.

### CODE REVIEW

File: p2p/simulations/adapters/types.go [hunk 1]
```diff
@@ -211,6 +228,7 @@ func RandomNodeConfig() *NodeConfig {
 		Name:            fmt.Sprintf("node_%s", enodId.String()),
 		Port:            port,
 		EnableMsgEvents: true,
+		LogVerbosity:    log.LvlUndefined,
```
Comment (fjl) [inline p2p/simulations/adapters/types.go:231]: This is the only place where LvlUndefined is used. Can't we just use 0 here?

Comment (rjl493456442) [reply p2p/simulations/adapters/types.go:231]: 0 is the critial. So I added this stuff.

Comment (rjl493456442) [reply p2p/simulations/adapters/types.go:231]: Or we can just pass the `log.LvlInfo` by default.

**Review by fjl — APPROVED** [review]
Changes look OK to me. I didn't test this, but all changes
here are in p2p/simulations, so it's non-critical. I trust that
you have tried this.

**Pull request merged**